```
../video/out/wayland_common.c:4883:9: error: implicit declaration of function 'wl_fixes_destroy'; did you mean 'wl_touch_destroy'? [-Wimplicit-function-declaration]
 4883 |         wl_fixes_destroy(wl->fixes);
```
Gonna have to guard that one too.

--- a/video/out/wayland_common.c
+++ b/video/out/wayland_common.c
@@ -4879,6 +4879,11 @@ void vo_wayland_uninit(struct vo *vo)
     if (wl->shm)
         wl_shm_destroy(wl->shm);
 
+#ifdef WL_FIXES_DESTROY
+    if (wl->fixes)
+        wl_fixes_destroy(wl->fixes);
+#endif
+
     if (wl->single_pixel_manager)
         wp_single_pixel_buffer_manager_v1_destroy(wl->single_pixel_manager);
 

